xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxChainingRenderer.java

```diff
@@ -704,7 +705,7 @@ protected void printParameters(Map<String, String> parameters, boolean newLine)
         }
 
         if (parametersStr.length() > 0) {
-            StringBuffer buffer = new StringBuffer("(%");
+            StringBuilder buffer = new StringBuilder("(%");     //Replaced StringBuffer with unsynchronized StringBuilder
```

Comment:
There is really no need for this comment.

Reply:
K

Reply:
Done.